addon/components/paper-ink-bar.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+const { computed } = Ember;
+
+export default Ember.Component.extend({
```

Comment:
Oh yes, the new `disallowDirectPropertyAccess` directive set to `true` by default now I believe.

Will update the code accordingly.